backend/src/test/java/com/google/sps/utils/TestUtils.java

```diff
@@ -100,4 +100,41 @@ public static Entity getUserEntityWithAllergiesOnly() {
     user.setProperty(UserConstants.PROPERTY_ALLERGIES, ALLERGIES);
     return user;
   }
+
+  public static Recipe getRecipe() {
+    String name = "Recipe";
+    String time = "10 min ";
+    String calories = "140 calories";
+    String difficulty = "Easy";
+    String imageUrl = "imageUrl";
+    List<String> diet = Collections.singletonList("vegetarian");
+    List<String> ingredients = Arrays.asList("3 lemons", "140g caster sugar", "1l cold water");
+    List<String> instructions =
+        Arrays.asList(
+            "Tip the lemons, sugar and half the water "
+                + "into a food processor and blend until the lemon is finely chopped.",
+            "Pour the mixture into a sieve over a bowl, then press through as much "
+                + "juice as you can. Top up with the remaining water and serve with plain "
+                + "ice or frozen with slices of lemon and lime.");
+    return new Recipe(name, time, calories, difficulty, imageUrl, diet, ingredients, instructions);
+  }
+
+  public static Entity getRecipeEntity() {
```

Comment:
nit: Can we call this createTestRecipeEntity() to clarify what the method is doing?